.travis.yml

```diff
@@ -2,7 +2,7 @@ sudo: false
 language: ruby
 cache: bundler
 before_install:
-  - gem update --system
+  - gem i rubygems-update -v '<3' && update_rubygems
```

Comment:
This change became necessary to avoid failure for Ruby 2.1 and 2.2:

```shell
$ gem update --system

Updating rubygems-update
Fetching: rubygems-update-3.0.4.gem (100%)

ERROR:  Error installing rubygems-update:
	rubygems-update requires Ruby version >= 2.3.0.

The command "gem update --system" failed and exited with 1 during .
```